cmd/root.go

```diff
@@ -31,9 +32,15 @@ const (
 	etcPivotFile          = "/etc/pivot/image-pullspec"
 	runPivotRebootFile    = "/run/pivot/reboot-needed"
 	// Pull secret.  Written by the machine-config-operator
-	kubeletAuthFile       = "/var/lib/kubelet/config.json"
+	kubeletAuthFile = "/var/lib/kubelet/config.json"
+	// File containing kernel arg changes for tuning
+	kernelTuningFile = "/etc/pivot/kernel-args"
 )
 
+// TODO: fill out the whitelist
+// tuneableArgsWhitelist contains allowed keys for tunable arguments
+var tuneableArgsWhitelist = []string{"nosmt"}
```

Comment:
I'd have this as a `map[string]bool` where in `isArgTunable` you can just do:

```go

var tuneableArgsWhitelist = map[string]bool{
    {"nosmt": true},
}

func isArgTunable(arg string) bool {
	return tuneableArgsWhitelist[arg]
}
```

Reply:
I like this. Will update.